src/pages/docs/administration/managing-infrastructure/server-configuration-and-file-storage/index.md

```diff
@@ -46,9 +46,11 @@ The Octopus Server stores files in the following folders by default:
     - This is where all performance and other temporal telemetry files are stored.
     - See this [page](/docs/administration/managing-infrastructure/server-configuration-and-file-storage/moving-octopus-server-folders/#MovingOctopusServerfolders-Telemetry) on how to move the Octopus Server telemetry folder
 - `C:\Octopus\Imports`
-    - This folder was added in **Octopus 2021.1**
```

Comment:
Since we decided to launch this when 2023.3 comes out, we've stripped all references to 2023.3 in this PR.

However, when I stripped the one here, I saw this old comment.

Since we keep our docs aligned with "the latest" version of the product, this comment is very old.

So I stripped it.

@felixi-octopus @harrisonmeister , let me know if you disagree with this choice.

Reply:
I think removing this is fine, it's ~2 years old now :smile: